service/paging_test.go

```diff
@@ -0,0 +1,79 @@
+package service
+
+import (
+	"net/http"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestUnit_ParsePagingParams(t *testing.T) {
+
+	type testCase struct {
+		name     string
+		request  func(t *testing.T) *http.Request
+		validate func(t *testing.T, req *http.Request)
+	}
+
+	tests := []testCase{
+		{
+			name: "happy path",
+			request: func(t *testing.T) *http.Request {
+				req, err := http.NewRequest("GET", "http://example.com?page_number=1&page_size=100&sort_asc=true&sort_field=foo", nil)
+				assert.Nil(t, err)
+				return req
+			},
+			validate: func(t *testing.T, req *http.Request) {
+				pagingParams, err := ParsePagingParams(req, []string{"foo"})
```

Comment:
super minor, would love if we had 1 other string in the whitelist

Reply:
good call.  will add.